src/Wizard.js

```diff
@@ -418,6 +400,30 @@ export default class Wizard extends Webform {
     this.allPages = allComponents;
   }
 
+  getSorteredComponents({ components, originalComponents }) { // sorts components if they were shuffled after the conditional logic
```

Comment:
Minor point, should this be "getSorted" rather than "getSortered"?

Reply:
Yes, certainly. Thank you